src/views/partials/header.hbs

```diff
@@ -0,0 +1,8 @@
+<div class="header">
```

Comment:
i feel like we are getting the same stuff done, so for example this header.hbs i made my own too but ill just use yours so we need to decide better whose working on what